exporter/exporter.go

```diff
@@ -21,11 +21,14 @@ import (
 	"context"
 	"fmt"
 	"net/http"
+	"os"
 	"sync"
 
-	"github.com/percona/exporter_shared"
+	"github.com/go-kit/log/level"
```

Comment:
Can we remove this dependency?
It is used only once so, I think it is better to keep 3rd party dependencies at minimum.

Thanks.

Reply:
updated :+1:

Reply:
Note this will still get pulled in as a transitive dependency of `prometheus/exporter-toolkit`